assets/app/views/_overview-deployment.html

```diff
@@ -106,7 +107,7 @@
           </a>
         </div>
 
-        <div column class="scaling-controls">
+        <div column class="scaling-controls" ng-if="!hpa">
```

Comment:
Because it didn't used to be an array and I forgot to change this reference.... Thanks.